src/openms/include/OpenMS/METADATA/Precursor.h

```diff
@@ -72,7 +72,7 @@ namespace OpenMS
       ECD,                      ///< Electron capture dissociation
       IMD,                      ///< Infrared multiphoton dissociation
       SORI,                     ///< Sustained off-resonance irradiation
-      HCID,                     ///< High-energy collision-induced dissociation
+      HCD,                     ///< High-energy collision-induced dissociation
```

Comment:
I remember there was some discussion around it... we might need to revisit that one again to be sure this is correct

Reply:
The mzmls that I used all had HCD instead of HCID... Should we keep both of them then?

Reply:
I think this was one part of the discussion: https://github.com/OpenMS/OpenMS/pull/2080

Reply:
I am inclined to just keep HCD as you proposed but I think there might be some issue with just those changes. If you search for HCID you see that this shows up a few times. e.g., https://github.com/OpenMS/OpenMS/blob/1db1c512204b7345cdd1ba0668b4efd6d9624164/src/tests/topp/THIRDPARTY/spectra_spectrast.mzXML#L20 has HCID in the mzXML file. Is this properly parsed then and stored as HCD?